<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Peter Simonsson</author>
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
</comment>
 <class>Konversation::ChannelOptionsUI</class>
 <widget class="QWidget" name="Konversation::ChannelOptionsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>421</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="channelTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="topicTab">
      <attribute name="title">
       <string>Topi&amp;c</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QSplitter" name="splitter4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="topicHistoryList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Nickname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Topic</string>
           </property>
          </column>
         </widget>
         <widget class="KTextEdit" name="topicPreview">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="KTextEdit" name="topicEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modesPage">
      <attribute name="title">
       <string>Mo&amp;des</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="topicModeChBox">
         <property name="text">
          <string>&amp;Topic can only be changed by channel operators</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="messageModeChBox">
         <property name="text">
          <string>&amp;No messages from outside the channel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="secretModeChBox">
         <property name="text">
          <string>&amp;Secret channel, the channel is not listed in the channel list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="inviteModeChBox">
         <property name="text">
          <string>Only &amp;invited are allowed to join the channel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="moderatedModeChBox">
         <property name="text">
          <string>&amp;Moderated channel, only people with voice can write to the channel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <item row="1" column="0">
          <widget class="QCheckBox" name="keyModeChBox">
           <property name="text">
            <string>Channel &amp;password:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KIntNumInput" name="userLimitEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="userLimitChBox">
           <property name="text">
            <string>User &amp;limit:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="spacer10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>211</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="KLineEdit" name="keyModeEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="toggleAdvancedModes">
           <property name="text">
            <string>Show &amp;Advanced Modes &gt;&gt;</string>
           </property>
           <property name="shortcut">
            <string>Alt+A, Backspace, Tab, Backspace</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>301</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="otherModesList">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>&amp;Ban List</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="K3ListViewSearchLine" name="banListSearchLine" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="K3ListView" name="banList">
         <property name="showSortIndicator">
          <bool>true</bool>
         </property>
         <property name="itemsRenameable">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Hostmask</string>
          </property>
          <property name="clickable">
           <bool>true</bool>
          </property>
          <property name="resizable">
           <bool>true</bool>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Set By</string>
          </property>
          <property name="clickable">
           <bool>true</bool>
          </property>
          <property name="resizable">
           <bool>true</bool>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time Set</string>
          </property>
          <property name="clickable">
           <bool>true</bool>
          </property>
          <property name="resizable">
           <bool>true</bool>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="removeBan">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Remove Ban</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addBan">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Add Ban</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <customwidgets>
  <customwidget>
   <class>K3ListView</class>
   <extends>Q3ListView</extends>
   <header>k3listview.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>K3ListViewSearchLine</class>
   <extends>QWidget</extends>
   <header>k3listviewsearchline.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ktextedit.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>keyModeChBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>keyModeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userLimitChBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>userLimitEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
